### PR TITLE
Fix cocoapods installation

### PR DESCRIPTION
**Pull request by luoxiu**
and we may have encountered a compiler bug

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,6 +44,9 @@ package.dependencies += [
 
 ```ruby
 pod 'CombineX', "~> 0.3.1"
+
+# or, if you want to use `Foundation` extensions: 
+pod 'CombineX/CXFoundation', "~> 0.3.1"
```
Comment (ddddxxx) [inline README.md:49]: I have very little knowledge of CocoaPods. So we don't need to state CXFoundation explicitly before, but now we do? With duplicate version number?

Comment (luoxiu) [reply README.md:49]: It's an either-or

`pod 'CombineX'` means you only want `CombineX`.
`pod 'CombineX/CXFoundation` means you want  `CombineX`, and `CXFoundation`.

Comment (ddddxxx) [reply README.md:49]: Did users need `pod 'CombineX/CXFoundation'` before?

Comment (luoxiu) [reply README.md:49]: Yes, if they want to use `CXFoundation`

**Review by ddddxxx — APPROVED** [review]
LGTM

**Pull request merged**